src/integration/pages/onDemandAudioPage/indonesia/canonical.test.js

```diff
@@ -4,11 +4,9 @@
  */
 
 import runCrossPlatformTests from '../crossPlatformTests';
-import runCanonicalTests from '../canonicalTests';
```

Comment:
Did we mean to delete these?

Reply:
I think we should move crossPlatform into canonical for consistency & to prevent confusion